From 354bbf9657385519dceeff6a890a71f9d53e6a8f Mon Sep 17 00:00:00 2001
From: Parth Gajjar <parth.gajjar@amd.com>
Date: Tue, 17 Jan 2023 02:04:32 -0800
Subject: [PATCH] libweston: GL_EXT_unpack_subimage not supported for GLES2.0

Removing GL_EXT_unpack_subimage as not supported by GLES2.0

Signed-off-by: Parth Gajjar <parth.gajjar@amd.com>
---
 libweston/renderer-gl/gl-renderer.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libweston/renderer-gl/gl-renderer.c b/libweston/renderer-gl/gl-renderer.c
index 613ddf4..046a26f 100644
--- a/libweston/renderer-gl/gl-renderer.c
+++ b/libweston/renderer-gl/gl-renderer.c
@@ -3941,11 +3941,13 @@ gl_renderer_setup(struct weston_compositor *ec, EGLSurface egl_surface)
 	else
 		ec->read_format = PIXMAN_a8b8g8r8;
 
+#if 0
 	if (gr->gl_version < gr_gl_version(3, 0) &&
 	    !weston_check_egl_extension(extensions, "GL_EXT_unpack_subimage")) {
 		weston_log("GL_EXT_unpack_subimage not available.\n");
 		return -1;
 	}
+#endif
 
 	if (gr->gl_version >= gr_gl_version(3, 0) ||
 	    weston_check_egl_extension(extensions, "GL_EXT_texture_type_2_10_10_10_REV"))
-- 
2.34.1

